db/views/holiday_usage_last_years_v01.sql

```diff
@@ -0,0 +1,23 @@
+SELECT latest_runs.id,
+       data.*
+FROM
+  (
+    SELECT alert_generation_run_id, school_id, data.*
+    FROM alerts, alert_types, jsonb_to_record(variables) AS data(
+      last_year_holiday_gas_gbp float,
+      last_year_holiday_electricity_gbp float,
+      last_year_holiday_gas_gbpcurrent float,
+      last_year_holiday_electricity_gbpcurrent float,
+      last_year_holiday_gas_kwh_per_floor_area float,
+      last_year_holiday_electricity_kwh_per_floor_area float,
+      name_of_last_year_holiday text
```

Comment:
probably need to change the alert here to have holiday type, start and end date like https://github.com/Energy-Sparks/energy-sparks/blob/b7522294e257fdccd83950e13824d1d79e08251f/db/views/electricity_consumption_during_holidays_v02.sql#L9 ?

Reply:
Ahh ok, didn't realise this was a thing, thanks

Reply:
Analytics doesn't have holiday_type available for this alert (it looks like it could have with a change though). Where did the requirement come from to add this extra info?

Reply:
I have just worked out that you had to add it to analytics.